docs/content/doc/features/comparison.zh-cn.md

```diff
@@ -62,7 +62,7 @@ _表格中的符号含义:_
 | Git LFS 2.0                              | ✓     | ✘    | ✓         | ✓         | ✓         | ⁄         | ✓            |
 | 组织里程碑                               | ✘     | ✘    | ✘         | ✓         | ✓         | ✘         | ✘            |
 | 细粒度用户角色 (例如 Code, Issues, Wiki) | ✓     | ✘    | ✘         | ✓         | ✓         | ✘         | ✘            |
-| 提交人的身份验证                         | ✘     | ✘    | ?         | ✓         | ✓         | ✓         | ✘            |
+| 提交人的身份验证                         | ⁄     | ✘    | ?         | ✓         | ✓         | ✓         | ✘            |
```

Comment:
What do it mean?

Reply:
`⁄`  means supported with limited functionality 

https://github.com/go-gitea/gitea/blob/b678d1adae759a6a5f3108b0d467d9fe79b6a19b/docs/content/doc/features/comparison.en-us.md?plain=1#L67